src/java.desktop/windows/classes/sun/awt/windows/ThemeReader.java

```diff
@@ -111,11 +121,16 @@ private static Long getTheme(String widget) {
             try {
                 if (!valid) {
                     // Close old themes.
-                    for (Long value : widgetToTheme.values()) {
-                        closeTheme(value);
+                    if (!dpiAwareWidgetToTheme.isEmpty()) {
```

Comment:
The `!isEmpty()` seems redundant. If the map is empty, `values()` returns empty set; iterating over empty set does nothing.